package.json

```diff
@@ -8,6 +8,7 @@
   },
   "dependencies": {
     "@claviska/jquery-minicolors": "2.3.6",
+    "@mcaptcha/vanilla-glue": "0.1.0-alpha-1",
```

Comment:
alpha version seems not very promising, but I guess we are the first user of this according to download counts on https://www.npmjs.com/package/@mcaptcha/vanilla-glue.

Reply:
@realaravinth mentioned that the glue code is already "stable". So it shouldn't be anything major to bump the version to a stable 1.0 version.

Reply:
The glue library has tests but I would like to wait for some deployment and see if any in-the-wild bugs are uncovered before I bump it to 1.0.

There won't be any changes in the library, it doesnt do much and what it does is simple enough to not require major changes.

At any rate, I will be available to handle support requests that are related to mCaptcha :)

Reply:
Okay, please make sure to outline any future breaking changes before 1.0.0 in the GitHub release notes or a CHANGELOG file please.